lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -126,3 +128,12 @@ function handleOnboardingManualRebase(onboardingPr: Pr): void {
     OnboardingState.prUpdateRequested = true;
   }
 }
+
+function invalidateExtractCache(baseBranch: string): void {
+  const cache = getCache();
+  cache.scan ||= {};
+
+  if (cache.scan[baseBranch]) {
```

Comment:
```suggestion
  if (cache.scan?.[baseBranch]) {
```
if we do this we don't need the scan set above.